pandas/tests/copy_view/test_indexing.py

```diff
@@ -919,3 +919,51 @@ def test_set_value_copy_only_necessary_column(
             assert not np.shares_memory(get_array(df, "a"), get_array(view, "a"))
         else:
             assert np.shares_memory(get_array(df, "a"), get_array(view, "a"))
+
+
+@pytest.mark.parametrize("dtype", ["int64", "Int64"])
+def test_where_noop(using_copy_on_write, dtype):
```

Comment:
I suppose you put it intentionally in test_indexing.py instead of test_methods.py? 
Personally I would still put it in test_methods.py, although it is an indexing like method, everything in this file are actual indexing operations.

Reply:
Yeah was intentional, but ok to move over. We will have to split them up anyway. Wanted to tackle this after 2.0